apps/ucp-ui/src/Institutions/InstitutionFilters.tsx

```diff
@@ -0,0 +1,183 @@
+        }}
+        label={INSTITUTIONS_FILTER_SEARCH_LABEL_TEXT}
+        onChange={({ target: { value } }) => {
+          handleChangeParams({
+            search: value,
+          });
+        }}
+        value={institutionsParams.search}
+      />
+      <FormGroup className={styles.formGroup}>
+        <FormHelperText>Aggregator</FormHelperText>
+        {isAggregatorsError && (
+          <Alert icon={false} severity="error">
+            <AlertTitle>Filter not found</AlertTitle>
+            {INSTITUTIONS_FILTER_AGGREGATORS_ERROR_TEXT}
+          </Alert>
+        )}
+        {isAggregatorsLoading && (
+          <>
+            <LoadingCheckbox />
+            <LoadingCheckbox />
+          </>
+        )}
+        {aggregatorsData?.aggregators?.map(({ displayName, name, id }) => (
+          <FormControlLabel
+            control={
+              <Checkbox
+                checked={institutionsParams.aggregatorName?.includes(name)}
+                onChange={createChangeAggregatorHandler(name)}
+              />
+            }
+            key={id}
+            label={displayName}
+            slotProps={slotProps}
+          />
+        ))}
+        <FormHelperText>Filter by Job Type</FormHelperText>
+        {jobTypeCheckboxes.map(({ label, prop }) => (
+          <FormControlLabel
+            control={
+              <Checkbox
+                checked={
+                  institutionsParams[prop as keyof InstitutionParamsBooleans]
+                }
+                onChange={({ target: { checked } }) => {
+                  handleChangeParams({
+                    [prop]: checked.toString(),
+                  });
+                }}
+              />
+            }
+            key={prop}
+            label={label}
+            slotProps={slotProps}
+          />
+        ))}
+        <FormHelperText>Other Features</FormHelperText>
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={institutionsParams.supportsOauth}
+              onChange={({ target: { checked } }) =>
+                handleChangeParams({
+                  supportsOauth: checked.toString(),
+                })
+              }
+            />
+          }
+          label={INSTITUTIONS_FILTER_OAUTH_LABEL_TEXT}
+          slotProps={slotProps}
+        />
+        <Divider />
+        <FormControlLabel
+          className={styles.inactiveSwitch}
+          control={
+            <Switch
+              checked={institutionsParams.includeInactiveIntegrations}
+              onChange={({ target: { checked } }) =>
+                handleChangeParams({
+                  includeInactiveIntegrations: checked.toString(),
+                })
+              }
+              size="small"
+            />
+          }
+          label={INSTITUTIONS_FILTER_INCLUDE_INACTIVE_INTEGRATIONS_LABEL_TEXT}
+          slotProps={slotProps}
+        />
+      </FormGroup>
+    </div>
+  );
+};
+
+export default InstitutionFilters;
```

Comment:
We previously decided to export inline, and not at the end of the file. Is that pattern different with Components?

Reply:
<img width="341" alt="image" src="https://github.com/user-attachments/assets/fbfbb56d-9265-407a-8e5e-2a7c1d974701">

I just do it so I can name the the default export, and vscode will pick it up easier

Reply:
Makes sense. Thanks for explaining.